src/App.tsx

```diff
@@ -1,41 +1,63 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
```

Comment:
create index.ts file in components folder so you, will be able to do like that
```suggestion
import { PostsList, PostDetails } from './components';
```